src/main/java/com/khutircraftubackend/auth/AuthResponseMessages.java

```diff
@@ -0,0 +1,18 @@
+package com.khutircraftubackend.auth;
+
+public final class AuthResponseMessages {
+
+    public static final String EMAIL_NOT_FOUND = "Користувач з таким email : %s не знайдений.";
+    public static final String EMAIL_BLOCKED = "Користувач з поштою %s заблокований.";
+    public static final String EMAIL_ENABLED = "Вітаємо з успішним входом.";
+    public static final String EMAIL_IS_ALREADY_IN_USE =
+            "Цей email зайнятий, використайте інший або ввійдіть під цим";
```

Comment:
for these 4 message constants only the last one `EMAIL_IS_ALREADY_IN_USE` is consistent with the meaning of its name. 
the rest three are a bit misleading.

`EMAIL_NOT_FOUND` better to rename into `USER_NOT_FOUND`
`EMAIL_BLOCKED` better to rename into `USER_IS_BLOCKED`
`EMAIL_ENABLED` better rename into `USER_LOGGED_IN`

Reply:
Yes, I agree, inattentive(